tutorials/napps/publishing_your_napp.rst

```diff
@@ -70,15 +63,15 @@ using your **username** (the one you have just registered).
    - at least three characters
   --------------------------------------------------------------
 
-The first question is related to your **username**. The second question is the
-name of your NApp, and, for now,
-we will just use ``my_first_napp`` as NApp name. The third question is related
+The first question is related to your **username** (username should be same as your github account username).
+The second question is the name of your NApp, and, for now,
+we will just use ``MyfirstNapp`` as NApp name. The third question is related
```

Comment:
@krishna4041 let's make this `MyfirstNapp` snake_case as the rest of the docs typically follows. Could you find all the occurrences and replace it with `my_first_napp`?